includes/Database_Upgrader.php

```diff
@@ -330,6 +333,32 @@ protected function yoast_reindex_stories() {
 		}
 	}
 
+	/**
+	 * Migration media post meta to taxonomy term.
+	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 *
+	 * @return void
+	 */
+	protected function migration_post_meta() {
```

Comment:
Nit: this name is a bit generic. Maybe `add_poster_generation_media_source` or something?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5635/commits/54a6709daa9550f25650c417f77f5b3d9ac12580